WordPress/gutenberg
# Elements: Update the load order of the CSS in the Site Editor

## What?
The CSS for elements needs to load in the same order in the Site Editor as it does in the frontend. In https://github.com/WordPress/gutenberg/pull/41981 we made a change so that the selectors are used in a deterministic order.

## Why?
This is important so that the styles load in the same order on the frontend and in the editor.

## How?
We use the order of the elements in the config.

## Testing Instructions
- In your theme.json set one color for your heading elements and another color for your H2 elements:
```
			"h2": {
				"color": {
					"text": "red"
				}
			},
			"heading": {
				"color": {
					"text": "blue"
				}
			},
```
- Add an H2 to a template in the site editor
- Save the change
- The H2 should be red in both the frontend and the editor.

@WordPress/block-themers
